### ISSUE TITLE
can't set/edit public SSHKEY on playground

### ISSUE THREAD

**Issue by xmonader** [threefoldtech/grid_weblets#1284]
I tried from an existing account to change the public ssh key and from a newly created account to set the public key and both scenarios give

![image](https://user-images.githubusercontent.com/64129/216820036-73f55a33-b6f9-4da5-bebd-86fea7c776d4.png)

**Comment by MohamedElmdary** [comment]
@xmonader @AhmedHanafy725  I have gif showing the entire workflow ... works just fine
https://user-images.githubusercontent.com/31689104/216821975-a39a5987-ed6c-4762-babe-33246cd5ecae.mp4

**Comment by A-Harby** [comment]
> I tried from an existing account to change the public ssh key 

I tried change it many times and haven't face any issue.
![image](https://user-images.githubusercontent.com/109473873/216824819-6a2de804-f755-47ff-b7f0-8b2ddf0c70b7.png)

> and from a newly created account to set the public key and both scenarios give

And once created a new account, you're required to fund the account to create or enter ssh key.
![image](https://user-images.githubusercontent.com/109473873/216824904-df13cbcb-59d4-4f62-a687-74ece88a02ca.png)
And after funding the twin I was able to create and change ssh key.
